commands.json

```diff
@@ -30,5 +288,878 @@
+              }
+            ]
+          },
+          {
+            "name": "tags",
+            "type": "block",
+            "optional": true,
+            "block": [
+              {
+                "name": "tags",
+                "type": "enum",
+                "enum": [
+                  "TAGS"
+                ]
+              },
+              {
+                "name": ["open", "close"],
+                "type": ["string", "string"]
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "name": "slop",
+        "type": "integer",
+        "optional": true,
+        "command":"SLOP"
+      },
+      {
+        "name": "tags",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "INORDER"
+        ]
+      },
+      {
+        "name": "language",
+        "type": "string",
+        "optional": true,
+        "command":"LANGUAGE"
+      },
+      {
+        "name": "expander",
+        "type": "string",
+        "optional": true,
+        "command":"EXPANDER"
+      },
+      {
+        "name": "scorer",
+        "type": "string",
+        "optional": true,
+        "command":"SCORER"
+      },
+      {
+        "name": "explainscore",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "EXPLAINSCORE"
+        ]
+      },
+      {
+        "name": "payload",
+        "type": "string",
+        "optional": true,
+        "command":"PAYLOAD"
+      },
+      {
+        "name": "sortby",
+        "type": "block",
+        "optional": true,
+        "block": [
+          {
+            "name": "sortby",
+            "type": "string",
+            "command":"SORTBY"
+          },
+          {
+            "name": "order",
+            "type": "enum",
+            "optional": true,
+            "enum": [
+              "ASC", "DESC"
+            ]
+          }
+        ]
+      },
+      {
+        "name": "limit",
+        "type": "block",
+        "optional": true,
+        "block": [
+          {
+            "name": "limit",
+            "type": "enum",
+            "enum": [
+              "LIMIT"
+            ]
+          },
+          {
+            "name": ["offset", "num"],
+            "type": ["integer", "integer"]
+          }
+        ]
+      }
+    ],
+    "since": "1.0.0",
+    "group": "search"
+  },
+
+  "FT.AGGREGATE": {
+    "summary": "Adds terms to a dictionary",
+    "complexity": "O(1)",
+    "arguments": [
+      {
+        "name": "index",
+        "type": "string"
+      },
+      {
+        "name": "query",
+        "type": "string"
+      },
+      {
+        "name": "verbatim",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "VERBATIM"
+        ]
+      },
+      {
+        "name": "load",
+        "type": "block",
+        "optional": true,
+        "block": [
+          {
+            "name": "count",
+            "type": "string",
+            "command":"LOAD"
```

Comment:
What about `LOAD *`? Do we want to add this option? (either `LOAD *` or `LOAD count field [field...]`)
Could be useful here, although currently they wouldn't be mutual exclusive.

Reply:
Let's wait with it to the new syntax with `oneof` option.